From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 18 Apr 2021 13:16:37 +0100
Subject: test robust expansion

Match fish{1,2}.png instead of fish[12].png since it's an error if the
file is not found.
---
 test/run_tests.bash | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/test/run_tests.bash b/test/run_tests.bash
index b87f45d..d31e965 100755
--- a/test/run_tests.bash
+++ b/test/run_tests.bash
@@ -77,7 +77,7 @@ fi
 "$pdiff" --help | grep -i 'usage'
 
 rm -f diff.png
-"$pdiff" --output diff.png --verbose fish[12].png 2>&1 | grep -q 'FAIL'
+"$pdiff" --output diff.png --verbose fish{1,2}.png 2>&1 | grep -q 'FAIL'
 ls diff.png
 rm -f diff.png
 
@@ -86,10 +86,10 @@ head fish1.png > fake.png
 rm -f fake.png
 
 mkdir -p unwritable.png
-"$pdiff" --output unwritable.png --verbose fish[12].png 2>&1 | grep -q 'Failed to save'
+"$pdiff" --output unwritable.png --verbose fish{1,2}.png 2>&1 | grep -q 'Failed to save'
 rmdir unwritable.png
 
-"$pdiff" fish[12].png --output foo 2>&1 | grep -q 'unknown filetype'
+"$pdiff" fish{1,2}.png --output foo 2>&1 | grep -q 'unknown filetype'
 "$pdiff" --verbose fish1.png 2>&1 | grep -q 'Not enough'
 "$pdiff" --down-sample -3 fish1.png Aqsis_vase.png 2>&1 | grep -q 'Invalid'
 "$pdiff" --threshold -3 fish1.png Aqsis_vase.png 2>&1 | grep -q 'Invalid'
@@ -97,7 +97,7 @@ rmdir unwritable.png
 "$pdiff" --verbose --scale fish1.png Aqsis_vase.png 2>&1 | grep -q 'FAIL'
 "$pdiff" --down-sample 2 fish1.png Aqsis_vase.png 2>&1 | grep -q 'FAIL'
 "$pdiff"  /dev/null /dev/null 2>&1 | grep -q 'Unknown filetype'
-"$pdiff" --verbose --sum-errors fish[12].png 2>&1 | grep -q 'sum'
+"$pdiff" --verbose --sum-errors fish{1,2}.png 2>&1 | grep -q 'sum'
 "$pdiff" --color-factor .5 -threshold 1000 --gamma 3 --luminance 90 cam_mb_ref.tif cam_mb.tif
 "$pdiff" --verbose -down-sample 30 -scale --luminance-only --fov 80 cam_mb_ref.tif cam_mb.tif
 "$pdiff" --fov wrong fish1.png fish1.png 2>&1 | grep -q 'Invalid argument'
